client/src/services/ApiClient.js

```diff
@@ -18,3 +19,22 @@ export async function getEmployeeByID(employeeID) {
       throw new Error("An error occurred during employee fetch");
   }
 }
+
+//CreatClient
+async function addClient(client) {
+ 
+  const response = await fetch(url + "/api/clients", {
+    method : 'POST',
+    headers: {'Content-Type': 'application/json'},
+    body: JSON.stringify({...client, fullName: client.firstName +" "+ client.lastName,
+     address: client.address +", "+ client.number+" "+ client.city+ ", "+client.postCode, wallet: 0})
+  });
+  
+
+  if(response.OK)
+  return null;
```

Comment:
try to format properly and maybe for handling response status you can use switch cases as we did as an example